src/test/java/it/gov/pagopa/reward/notification/repository/RewardOrganizationImportsRepositoryExtendedImplTest.java

```diff
@@ -0,0 +1,84 @@
+package it.gov.pagopa.reward.notification.repository;
+
+import it.gov.pagopa.reward.notification.BaseIntegrationTest;
+import it.gov.pagopa.reward.notification.dto.controller.ImportFilter;
+import it.gov.pagopa.reward.notification.enums.RewardOrganizationImportStatus;
+import it.gov.pagopa.reward.notification.model.RewardOrganizationImport;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.data.domain.Pageable;
+import org.springframework.util.ObjectUtils;
+
+import java.time.LocalDateTime;
+import java.util.List;
+
+class RewardOrganizationImportsRepositoryExtendedImplTest extends BaseIntegrationTest {
+
+    public static final LocalDateTime TODAY = LocalDateTime.now();
+    public static final String TEST_IMPORT_FILE_PATH = "test/import";
+    public static final String TEST_INITIATIVE_ID = "TEST_INITIATIVE";
+    public static final String TEST_ORGANIZATION_ID = "TEST_ORGANIZATION";
+
+    private static final RewardOrganizationImport testImport = RewardOrganizationImport.builder()
+            .filePath(TEST_IMPORT_FILE_PATH)
+            .initiativeId(TEST_INITIATIVE_ID)
+            .organizationId(TEST_ORGANIZATION_ID)
+            .feedbackDate(TODAY)
+            .eTag("ETAG")
+            .contentLength(5)
+            .url("URL")
+            .elabDate(TODAY)
+            .status(RewardOrganizationImportStatus.COMPLETE)
+            .errors(List.of(new RewardOrganizationImport.RewardOrganizationImportError()))
+            .build();
+
+    @Autowired
+    private RewardOrganizationImportsRepository importsRepository;
+    @Autowired
+    private RewardOrganizationImportsRepositoryExtendedImpl extendedRepository;
+
+    @BeforeEach
+    void prepareTestData() {
+        importsRepository.save(testImport).block();
```

Comment:
when building test data, store also at least one record that you should not expect as result.
This way you could verify that the filters are working as expected